### PR TITLE
Added pre-commit jobs for glance operator

### PR DESCRIPTION
**Pull request by amolkahat**
This patch adds jobs for glance and operator.

Signed-off-by: Amol Kahat <amolkahat@gmail.com>

### CODE REVIEW

**Comment by amolkahat** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-glance-operator-master-precommit-check

**Comment by amolkahat** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-glance-operator-master-precommit-check

**Comment by amolkahat** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-glance-operator-master-precommit-check

**Comment by amolkahat** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-glance-operator-master-precommit-check

**Comment by amolkahat** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-glance-operator-master-precommit-check

**Comment by amolkahat** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-glance-operator-master-precommit-check

File: ci-operator/config/openstack-k8s-operators/glance-operator/openstack-k8s-operators-glance-operator-master.yaml [hunk 1]
```diff
@@ -74,6 +79,20 @@ tests:
   optional: true
   steps:
     workflow: openstack-k8s-operators-test-build-deploy
+- as: precommit-check
+  commands: |
```
Comment (viroel) [inline ci-operator/config/openstack-k8s-operators/glance-operator/openstack-k8s-operators-glance-operator-master.yaml:83]: @bshewale when I test in my machine, only works if:
`export GOFLAGS=`

not sure if works for you without that.

**Comment by viroel** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-glance-operator-master-precommit-check

**Review by viroel — APPROVED** [review]

**Pull request merged**